### PR TITLE
Upgrade

### PR DESCRIPTION
**Pull request by JakeKo**
The code base is now compatible with OpenCV v3.3.0.

### CODE REVIEW

**Comment by dteske25** [comment]
Also, how does it feel to have the first PR in this repo?

**Comment by JakeKo** [comment]
It feels a little like I buffed a turd so it shines a little, presented it to the community as a joke, and they all applauded.

File: detection/detect_objects.cpp [hunk 1]
```diff
@@ -36,17 +37,17 @@ void filterLargest(vector<Circle>* key_circ, vector<Circle>* circ, int max_count
 }
 
 void detectObjects(Mat* frame, vector<Point2f>* centers, int obj_count = 5) {
-	gpu::GpuMat d_frame;
+	cuda::GpuMat d_frame;
 	Mat h_frame, h_red_blobs, h_lwr_red_blobs, h_upr_red_blobs, h_grn_blobs;
 	Scalar lowest_red(0, 150, 150), lower_red(20, 255, 255);
 	Scalar upper_red(160, 150, 150), uppest_red(180, 255, 255);
 	Scalar lower_grn(50, 80, 80), upper_grn(90, 255, 255);
```
[the remark this answers is not in the document]
Comment (JakeKo) [reply detection/detect_objects.cpp]: I'm just specifying one green range here. Basically it goes:
Range 1: lowest_red to lower_red
Range 2: upper_red to uppest_red
Range 3: lower_grn to upper_grn

Eventually, I'll get around to shifting the image color so we only need one range for red.

File: detection/do_test.cpp [hunk 2]
```diff
@@ -1,9 +1,7 @@
 #include "detect_objects.hpp"
 #include <cstdio>
-#include <opencv2/highgui/highgui.hpp>
-#include <opencv2/imgproc/imgproc.hpp>
-#include <opencv2/objdetect/objdetect.hpp>
-#include <opencv2/opencv.hpp>
+
+#define VID_PATH "/uav_rsc/arena-center2017.mp4"
```
[the remark this answers is not in the document]
Comment (JakeKo) [reply detection/do_test.cpp]: This is supposed to resemble default behavior - so if you run it with no CLI params it "just works." I can move this away from a definition and just make it check if a file path was passed. If not, plug in this file path.

**Pull request merged**